ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -221,6 +222,33 @@ export const displayTierBadgeByRequiredModule = (
   );
 };
 
+export const getEraNote = (profile: Profile): string => {
+  // prettier-ignore
```

Comment:
please remove this

Reply:
Removed. However, I disagree with prettier format. This looks bad to me as it makes code less readable.

```
const tierDisplayNames: AccessTierDisplayNames[] = Object.keys(
    AccessTierDisplayNames
  ).map((key) => {
    return AccessTierDisplayNames[key];
  });
  ```

Reply:
Yeah, I hear you.  I don't like all of Prettier's choices.  But I think it's the right choice to follow it in general.

BTW: if a block inside { } has only one statement, you can remove the { } and the `return`.  

Sometimes I like to play with different arrangements to see what looks best.  Here I might try something like:
```
const displayNamesKeys =  Object.keys(AccessTierDisplayNames);
const tierDisplayNames = displayNamesKeys.map((key) => AccessTierDisplayNames[key]);
```

But do whatever you think looks best and is Prettier-compatible.

Reply:
[update: please read the longer note below, first: https://github.com/all-of-us/workbench/pull/6240#discussion_r798659447]